fixtures/response/room_response_fixture.py

```diff
@@ -7,7 +7,6 @@
         response{
             responseId,
             createdDate,
-            missingItems,
```

Comment:
hey @KMaina, Kindly explain why this is being removed?

Reply:
@RachaelNantale this is being removed because the response is querying for missing resources in the room. My task was to ensure that a resource is not added to a room upon adding to the database. Thus this functionality should not exist.

Reply:
oh okay Thanks for the clarification @KMaina